### ISSUE TITLE
Facilitate conversion from CollectionFS

### ISSUE THREAD

**Issue by hluz** [jalik/jalik-ufs-gridfs#3]
https://github.com/jalik/jalik-ufs/issues/14

Making the gridfs collections compatible with the structure used by CFS would be great... At least the capability to store user fields (as in metadata) and using same format for _id (obj instead of meteor string) would make it possible to point at the same gridfs collection from this package and from CFS...

Many thanks for considering.

**Comment by jalik** [comment]
@hluz you can store any "meta" data directly in the file collection, if you need to store **userId** for example, just add this attribute to the document before inserting/updating. You just have to take care of not modifying attributes used by ufs like "complete", "uploading", "url"...

**Comment by hluz** [comment]
Yes, I understand. It is just that cfs:gridfs also adds the metadata object to the gridfs.files, not only to the file collection...  Also using Mongo.ObjectId() for _id would be similar format to cfs:gridfs, making it easier to migrate between the two without having to do any data conversion.

**Comment by rodrigok** [comment]
@hluz I didn't have time now, will be great if you can send us a PR with your ideas.

**Comment by krishaamer** [comment]
Great idea! Has any progress been made on this thus far @hluz @jalik @rodrigok ?

**Comment by jalik** [comment]
@krishaamer no progress on my side since I am focusing on the main/core UploadFS package. I know this request potentially concerns a lot of users but it takes time to create a migration tool.
